ote/src/cljc/ote/time.cljc

```diff
@@ -24,6 +25,24 @@
          cljs-time/to-default-time-zone
          (format/unparse (format/formatter "dd.MM.yyyy HH:mm"))))))
 
+#?(:cljs
+   (defn js-time-to-string [time]
```

Comment:
Isn't this formatting? Should it be called `format-js-time` or something like that. Or have a function `from-js-time` that turns JS Date object to OTE Time record and call format on that.

Reply:
We already have time/parse-time that converts string to ->Time object.
So I only change function name to format-js-time.